lib/trento_web/router.ex

```diff
@@ -183,12 +183,23 @@ defmodule TrentoWeb.Router do
         get "/activity_log", SettingsController, :get_activity_log_settings
 
         if Application.compile_env!(:trento, :suse_manager_enabled) do
-          scope "/suma_credentials" do
-            resources "/", SUMACredentialsController,
-              only: [:show, :create, :update, :delete],
-              singleton: true
+          scope "/suse_manager" do
+            get "/", SettingsController, :get_suse_manager_settings
+            post "/", SettingsController, :save_suse_manager_settings
+            patch "/", SettingsController, :update_suse_manager_settings
+            put "/", SettingsController, :update_suse_manager_settings
+            delete "/", SettingsController, :delete_suse_manager_settings
+            post "/test", SettingsController, :test_suse_manager_settings
+          end
 
-            post "/test", SUMACredentialsController, :test
+          # deprecated
```

Comment:
It's a pity that we cannot really deprecate the whole endpoint in openapi itself.
This way, the path is not shown as deprecated.
I have looked a bit, and didn't find any function to do so, without deprecating the new one as well